### DISCUSSION TITLE
How to fully uninstall Zen on MacOS?

### DISCUSSION THREAD

**Discussion by FrancisMorelli** [zen-browser/desktop#3651] [Q&A]
Hello.

I tried to uninstall Zen Browser with AppCleaner in order to have a fresh install, but it did work as I had expected.
When I installed Zen again, all essentials, pin tabs, opened tabs and color showed up as before.

I attached here a screenshot of what AppCleaner finds when I ask it to remove Zen.
Does anyone know how to fully uninstall Zen on MacOS?

<img width="428" alt="SCR-20241212-jxzk" src="https://github.com/user-attachments/assets/a1dbc25f-281d-4960-b086-573bf3eb5f93" />

**Comment by vraravam** [comment]
These seem valid.

Most likely your settings (essentials, pinned tabs, etc) might have been backed up to your Mozilla profile, and thus got resurrected. Can you please try and confirm?

**Reply by FrancisMorelli** [reply]
@vraravam, I disagree about it, because I have not set up my Mozilla account on Zen Browser. So, there is no way for this information to be retrieved from a backup.

**Comment by JSGRD22** [comment]
Zen, like any other browser, stores your user data in a folder called "Profiles", which contains everything from addons data to cookies to css styles. To locate it on your operating system, go to `about:profiles` and go to the root directory of the profile you are using.

**Reply by FrancisMorelli** [reply]
It did work! Thank you.
